utility/CalculateInstalledMemory.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ *   Copyright 2016 Pivotal Software, Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ **/
+
+#include "utility/CalculateInstalledMemory.hpp"
+#include "utility/UtilityConfig.h"
+
+#if defined(QUICKSTEP_HAVE_SYSCONF)
+#include <unistd.h>
+#elif defined(QUICKSTEP_HAVE_WINDOWS)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
+
+#include "glog/logging.h"
+
+namespace quickstep {
+namespace utility {
+namespace system {
+
+bool calculateTotalMemoryInBytes(std::uint64_t &total_memory) {
```

Comment:
We have to change `std::uint64_t &total_memory` to `std::uint64_t *total_memory`. Otherwise, `cpplint` will fail.

```
./utility/CalculateInstalledMemory.cpp:33:  Is this a non-const reference? If so, make const or use a pointer: std::uint64_t &total_memory  [runtime/references] [2]
```